src/seml/database.py

```diff
@@ -88,6 +104,11 @@ def get_mongodb_config(path=SETTINGS.DATABASE.MONGODB_CONFIG_PATH):
     database: <database_name>
     host: <host>
     directConnection: <bool> (Optional)
+    ssh_config: <dict> (Optional)
+      ssh_address_or_host: <the url of the jump host>
+      ssh_pkey: <the ssh host key>
+      ssh_username: <username for jump host>
+      ** further arguments passed to `SSHTunnelForwarder` (see https://github.com/pahaz/sshtunnel)
```

Comment:
Q: This will respect the ssh config, right?

Reply:
Not sure what you mean. But I would say yes, other elements in `ssh_config` will be passed to the TunnelForwarder